src/transport/emitter.ts

```diff
@@ -97,7 +88,7 @@ export class Emitter extends EventEmitter {
    * @param {Function} listener to call on event
    * @return {void}
    */
-  static on(event: "cloudevent" | "newListener" | "removeListener", listener: (...args: any[]) => void): void {
+  static on(event: string, listener: (...args: any[]) => void): void {
     this.getInstance().on(event, listener);
```

Comment:
While rereading the source code, I think it would be more consistent to use `Emitter.getInstance()` on this line too